### PR TITLE
Adding a tutorial on how to integrate UMH into Tuli[

### PR DESCRIPTION
**Pull request by JeremyTheocharis**
# Description

This 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

will be tested once PR is created by checking the auto-generated website on Netlify

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked that my issue complies with the [Contributing guidelines](https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+## Step-by-step tutorial
+
+{{% alert title="Info" color="primary" %}}
+See also Tulip knowledge base resource article [How to use the Machine Attributes API](https://support.tulip.co/en/articles/5007794-how-to-use-the-machine-attributes-api)
+{{% /alert %}}
+
+Perform the following steps as a Tulip administrator. First, in the Tulip platform, go to your profile icon and click Settings. Then select the Bot tab and create one. For this project, only data should be read and displayed in the Tulip environment, therefore the scope must be set to "read-only".
+ 
+{{< imgproc Picture2.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:43]: This is quite difficult to read, maybe provide a higher resolution base image ?

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+API Key, secret and authentication header is shown.
+{{< /imgproc >}}
+
+Then you will need to link this Machine API to a Machine.
+
+### Create a Machine Type
+
+Navigate to the tab “Shopfloor”  “Machines” then change to the tab “Machine Types”. Here, click in the button “Create type”. For now, only change the name of the machine type. 
+ 
+{{< imgproc Picture5.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md]: Same as with Image 2

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Creating a Machine.
+{{< /imgproc >}}
+
+### Add Machine Attribute
+
+Return to the Machine Type tab and navigate to the Machine Type you previously created. Click the edit button and scroll down and head to “Add Machine Attribute”.
+ 
+{{< imgproc Picture6.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:75]: Same as with Image 2

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Here you are able to create new machine attributes or assign ones you created and used before.
+ 
+{{< imgproc Picture7.png Fit "800x500" >}}
+Assign a name and a type to the Machine Attribute.
+{{< /imgproc >}}
+
+Having performed the previous steps, you are now able to to find the field mappings for each Machine Attribute. Go to the tab “Shopfloor/Machines”, select the machine name of interest, and click in the Configuration tab. There you’ll discover field mappings for each Machine Attributes. The “Attibute ID” and “Machine ID” will be used for setting up the connection in Node-RED.
+ 
+{{< imgproc Picture8.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:87]: What is this image for ?

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+In the manufacturer’s environment, UMH installed an electrical cabinet (E-rack) with an Industrial PC (IPC). The IPC was connected to the industrial controller of the machine and reads several data tags such as power time, hydraulic pump time and power consumption, with a flow in Node-RED (see also the blog article mentioned at the beginning of the article)
+ 
+{{< imgproc Picture9.png Fit "800x500" >}}
+Automatic saw with E-rack.
+{{< /imgproc >}}
+
+To connect to the Tulip Machine Attributes API, two nodes were used, namely a function node and a HTTP response. 
+ 
+{{< imgproc Picture10.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:103]: Same as with Image 2

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+Function and HTTP response nodes used.
+{{< /imgproc >}}
+
+For the function node, the message must contain the “Attribute ID” and “Machine ID” corresponding to the data tag being acquired from the industrial controller.
+ 
+{{< imgproc Picture11.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:109]: Same as with Image 2

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+{{< imgproc Picture12.png Fit "800x500" >}}
+HTTP response node configuration.
+{{< /imgproc >}}
+
+Having set everything up, the connection between the Node-RED flow and the Tulip platform is now established. On the next and final part of the blog, it will be shown how to use the data for developing it in the Tulip app environment.
+
+### Using the data in Tulip app
+Data coming from industrial controllers or sensors can be connected, leveraging the Machine Attribute created as any other Tulip device. For this, please navigate to the “Apps” tab and create a new app or select one you already created.
+
+{{< imgproc Picture13.png Fit "800x500" >}}
```
Comment (Scarjit) [inline docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:124]: Same as with Image 2

File: docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md (hunk 4 above)
Comment (JeremyTheocharis) [reply docs/content/en/docs/Tutorials/tulip/connecting-tulip-with-umh/index.md:87]: fixed by reordering the pictures

**Review by Scarjit — APPROVED** [review]

**Pull request merged**